### PR TITLE
Updated submodule link

### PR DESCRIPTION
**Pull request by Xierumeng**
From SSH to HTTPS so that credentials are not required for submodule checkout.

### CODE REVIEW

File: flutter_app/pubspec.lock [hunk 1]
```diff
@@ -61,10 +61,10 @@ packages:
     dependency: transitive
     description:
       name: collection
-      sha256: "4a07be6cb69c84d677a6c3096fcf960cc3285a8330b4603e0d463d15d9bd934c"
+      sha256: f092b211a4319e98e5ff58223576de6c2803db36221657b46c82574721240687
```
Comment (mgupta27) [inline flutter_app/pubspec.lock:64]: I think you forgot the quotation marks here.

File: flutter_app/pubspec.lock [hunk 2]
```diff
@@ -423,6 +423,14 @@ packages:
       url: "https://pub.dev"
     source: hosted
     version: "1.1.0"
+  web:
+    dependency: transitive
+    description:
+      name: web
+      sha256: dc8ccd225a2005c1be616fe02951e2e342092edf968cf0844220383757ef8f10
```
Comment (mgupta27) [inline flutter_app/pubspec.lock:430]: Quotation marks here as well. Just to maintain consistency.

**Review by mgupta27 — COMMENTED** [review]
Reviewed.

File: flutter_app/pubspec.lock (hunk 1 above)
Comment (Xierumeng) [reply flutter_app/pubspec.lock:64]: This is an automatically generated file.

File: flutter_app/pubspec.lock (hunk 2 above)
Comment (Xierumeng) [reply flutter_app/pubspec.lock:430]: This is an automatically generated file.

**Review by mgupta27 — APPROVED** [review]
Approved.

**Pull request merged**